tests/keras/backend/backend_test.py

```diff
@@ -1198,9 +1198,7 @@ def test_random_normal(self):
 
             # test that random_normal also generates different values when used within a function
             r = K.random_normal((1,), mean=mean, stddev=std, seed=1337)
-            samples = [K.eval(r) for _ in range(60000)]
-            assert np.abs(np.mean(samples) - mean) < std * 0.015
-            assert np.abs(np.std(samples) - std) < std * 0.015
+            assert abs(K.eval(r)[0] - K.eval(r)[0]) > 0.015 * std
```

Comment:
The reason for doing 60000 times is to test random distribution. IMHO, maybe we can make these evals in parallel or reduce 60000 to a reasonable number?

Reply:
I would argue that we already test the random distribution a few lines above: 
```python
rand = K.eval(K.random_normal((300, 200), mean=mean, stddev=std, seed=1337))
assert rand.shape == (300, 200)
assert np.abs(np.mean(rand) - mean) < std * 0.015
assert np.abs(np.std(rand) - std) < std * 0.015
```

Reply:
They are similar but somehow different. One is testing randomness across dimension. The other is testing randomness across time.

Reply:
Is this necessary? This piece of code was added to solve #10594. The proposed version also solve #10594  .

Reply:
That's how unit test works, we solve a bug and add a unit test to prevent it happens again.

Reply:
I got it, you are right. You are testing that it won’t repeat same value between consecutive two outcomes. I agree it’s able to catch this bug. Somehow I am talking about more regorious test. Sorry for my misunderstanding.

Reply:
No worries. It's true that this PR has drawbacks, so the choice is not obvious. I'll try to make a small summary pros/cons below to make it easier for a keras member to decide. Feel free to add your thoughts to it.

Reply:
The correct fix IMO is to draw a large array of values, a small amount of times. This tests across time, with statistical significance.

Something like `samples = [K.eval(K.random_normal((10, 10), ...)) for _ in range(10)]`